helevents/templates/rest_framework/event_list.html

```diff
@@ -354,6 +354,39 @@ <h3 id="enrolment">Open enrolment</h3>
         <pre><code>event/?enrolment_open_waitilist=true</code></pre>
         <p><a href="?enrolment_open_waitilist=true" title="json">See the result</a></p>
 
+        <h3 id="attendee-capacity">Attendee capacity</h3>
+        <p>Filters for filtering by event maximum_attendee_capacity and minimum_attendee_capacity:</p>
+
+        <h4 id="">Filtering for maximum_attendee_capacity</h4>
+        <p>It is possible to filter by maximum_attendee_capacity using gte (>) or lte (<) filters.</p>
+
+        <p><code>max_attendee_capacity_gte</code> parameter displays events with maximum_capacity greater than
+            or equal the applied parameter (integer value)</p>
+        <p>Example:</p>
+        <pre><code>event/?max_attendee_capacity_gte=10</code></pre>
+        <p><a href="?max_attendee_capacity_gte=10" title="json">See the result</a></p>
+
+        <p><code>max_attendee_capacity_lte</code> parameter displays events with maximum_capacity less than
```

Comment:
No such thing as "maximum_capacity". Either use:
```suggestion
        <p><code>max_attendee_capacity_lte</code> parameter displays events with maximum attendee capacity less than
```
or
```suggestion
        <p><code>max_attendee_capacity_lte</code> parameter displays events with <code>maximum_attendee_capacity</code> less than
```